src/java.base/share/classes/java/io/BufferedOutputStream.java

```diff
@@ -90,12 +86,9 @@ private BufferedOutputStream(OutputStream out, int initialSize, int maxSize) {
         }
 
         if (getClass() == BufferedOutputStream.class) {
-            // use InternalLock and resizable buffer when not sub-classed
-            this.lock = InternalLock.newLockOrNull();
-            this.buf = new byte[initialSize];    // resizable
+            // resizable if initialSize < maxSize
```

Comment:
I think the comment is confusing, can it just say resizable when not sub-classed as it did previously?

Reply:
Changed in 15bf969.